builtin/providers/aws/resource_aws_spot_fleet_request.go

```diff
@@ -275,13 +275,20 @@ func resourceAwsSpotFleetRequest() *schema.Resource {
 				Optional: true,
 				ForceNew: true,
 			},
+			"spot_request_state": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"client_token": &schema.Schema{
```

Comment:
Is this a thing that is supposed to be part of the schema? I thought it was "private" to TF?